[2024-03-18T21:28:54.891+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.Options_Crawler manual__2024-03-18T21:28:52.574270+00:00 [queued]>
[2024-03-18T21:28:54.897+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.Options_Crawler manual__2024-03-18T21:28:52.574270+00:00 [queued]>
[2024-03-18T21:28:54.898+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-18T21:28:54.907+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): Options_Crawler> on 2024-03-18 21:28:52.574270+00:00
[2024-03-18T21:28:54.912+0000] {standard_task_runner.py:60} INFO - Started process 69 to run task
[2024-03-18T21:28:54.915+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Options_Crawler', 'manual__2024-03-18T21:28:52.574270+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/crawler.py', '--cfg-path', '/tmp/tmpv92q0nv4']
[2024-03-18T21:28:54.917+0000] {standard_task_runner.py:88} INFO - Job 64: Subtask Options_Crawler
[2024-03-18T21:28:54.954+0000] {task_command.py:423} INFO - Running <TaskInstance: my_dag.Options_Crawler manual__2024-03-18T21:28:52.574270+00:00 [running]> on host 8aebcf34bdcd
[2024-03-18T21:28:55.016+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='Options_Crawler' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T21:28:52.574270+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-18T21:28:52.574270+00:00'
[2024-03-18T21:28:58.602+0000] {logging_mixin.py:188} INFO - ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
[2024-03-18T21:28:58.602+0000] {logging_mixin.py:188} INFO -  the error acuire in crawl data from TSETMS Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed'}
[2024-03-18T21:28:58.602+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-18T21:28:58.615+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Options_Crawler, execution_date=20240318T212852, start_date=20240318T212854, end_date=20240318T212858
[2024-03-18T21:28:58.662+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-18T21:28:58.681+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
